binderhub/static/index.js

```diff
@@ -121,6 +121,11 @@ function rstBadge(url) {
   return '.. image:: ' + BADGE_URL + ' :target: ' + url
 }
 
+function basicUrl(url) {
```

Comment:
I think we should just use the constant directly rather than adding a wrapper method.

Reply:
Unless this function was supposed to do more?